### ISSUE TITLE
New 3ds from black Friday bricked

### ISSUE THREAD

**Issue by jwin3** [AuroraWright/SafeA9LHInstaller#41]
Seems like I'm not the only one, but I followed plaillects guide to the T and got a bricked. Used the otpless safea9lh version 2.6.7. Tried all the troubleshooting steps and still on the black screen. I've done 4 o3ds without issue, so I think I know what I'm doing. I used a 64gb Samsung evo microsd card that was formatted properly before the system transfer took place. In gonna attempt the hard mod to restore my nand and will try the otpless again to see if I brick again. If I fail I'll downgrade and try to do the otp method.

**Comment by Morazotti** [comment]
Many people are having bricks with OTPless method.

**Comment by kkolat** [comment]
Which hax or entrypoint did you use to enter the homebre launcher ?

**Comment by jwin3** [comment]
I used ninjhax since i have a copy of cubic ninja. I formatted the sd card using the guiformat app listed in plailect guide with 32k sector.

**Comment by Misledz** [comment]
The black screen is something that I encountered recently, I thought it was a brick, turns out it was because of menuhax, just had to boot it up without the sd card, change theme and it was fine. Unless you stuck to ninjhax the whole procedure

**Comment by jwin3** [comment]
Menuhax was not the issue, still black screen.

**Comment by pbanj** [comment]
Do you have an and backup?

**Comment by jwin3** [comment]
yes i have a nand backup, when i get a chance (hopefully this Tuesday) I'm gonna try the hard mod and do the OTPless again.

**Comment by simon66** [comment]
"I used a 64gb Samsung evo microsd card that was formatted properly before the system transfer took place."

This makes me think the brick was caused from the **otp.bin** file located in "files9". When you do a system transfer, you have to delete the otp.bin from files9 (if exists) or else safeA9LH will use it by default, thus installing an otp.bin from a different system to your console (Which will cause a brick). 
@AuroraWright should probably implement something like this ---> #43 

Let us know if the Target SD card has the otp.bin file. If so, that might have caused your brick.
